http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -527,6 +529,43 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift: Parallel](
     }
   }
 
```

Comment:
I was assuming we're adding these `*InGoogle` methods instead of reusing the previously existing counterparts because we don't want to perform the DB operations for `postgress disk`. If so, I wonder if we should add a comment to the `*InGoogle` methods explaining why they exist and perhaps suggest they be removed when we start persisting `postgress disk` info.

Reply:
I was thinking the same - I planned on picking up the disk work post-alpha but in case that doesn't end up happening then comments for these methods are definitely very useful

Reply:
Maybe could even name this method `createGalaxyPostgresDiskOnlyInGoogle`?